site/src/render.tsx

```diff
@@ -27,7 +27,7 @@ const skuRender: Render<RenderContext> = {
     const versionMap = await braidVersionToDate();
 
     const sourceUrlPrefix = `${githubUrl}${prSha || 'master'}`;
-    const routerBasename = isGithubPages ? 'braid-design-system' : '';
+    const routerBasename = isGithubPages ? 'braid-src/lib/components' : '';
```

Comment:
This affects the docs site's page URLs so I don't think it should change. Guessing this was caught by the find and replace.